doc/source/timeseries.rst

```diff
@@ -360,37 +351,36 @@ Index constructor and pass in a list of datetime objects:
    index = pd.Index(dates)
    index
 
-Practically, this becomes very cumbersome because we often need a very long
+In practice this becomes very cumbersome because we often need a very long
 index with a large number of timestamps. If we need timestamps on a regular
-frequency, we can use the pandas functions ``date_range`` and ``bdate_range``
-to create timestamp indexes.
+frequency, we can use the ``date_range`` and ``bdate_range`` functions
+to create a ``DatetimeIndex``. The default frequency for ``date_range`` is a
```

Comment:
can you make date_range and bdate_range references to their docstring? (``:func:`date_range` ``)